Library/Homebrew/utils.rb

```diff
@@ -240,19 +240,19 @@ def install_gem!(name)
     odie "Failed to install/update the '#{name}' gem." if exit_code.nonzero?
   end
 
-  def install_gem_setup_path!(name)
+  def install_gem_setup_path!(name, executable: nil)
     install_gem!(name)
 
-    return if which(name)
+    return if which(executable)
```

Comment:
Do you want `executable` to default to `name` here as it was before, or default to `nil`?

Reply:
yes, whoops will fix above